### PR TITLE
[MRG] Minor pep8 fixes in docsim.py

### PR DESCRIPTION
**Pull request by dsquareindia**
Minor changes in docsim.py to make it more pep8 compliant. Left some places where I felt readability would suffer.

### CODE REVIEW

File: gensim/similarities/docsim.py [hunk 1]
```diff
@@ -104,7 +113,8 @@ def __getstate__(self):
         return result
 
     def __str__(self):
-        return ("%s Shard(%i documents in %s)" % (self.cls.__name__, len(self), self.fullname()))
+        return ("%s Shard(%i documents in %s)" % (self.cls.__name__, len(self),
```
Comment (piskvorky) [inline gensim/similarities/docsim.py]: We don't use hanging indent. It makes code harder to read (too cramped).

**Comment by piskvorky** [comment]
Thanks. We don't follow the 80-char guideline religiously, so I'm not sure we need this. But I guess it doesn't hurt either.

The vertically aligned indent changes we definitely don't want (use normal indent, no args on first line).

File: gensim/similarities/docsim.py (hunk 1 above)
Comment (dsquareindia) [inline gensim/similarities/docsim.py]: Oh alright sorry for that! I'll make the changes.

**Comment by dsquareindia** [comment]
I'll close this. Thanks for informing me about the conventions!

**Pull request closed without merging**